atomic_reactor/tasks/binary_container_build.py

```diff
@@ -103,23 +104,32 @@ def execute(self) -> Any:
             podman_remote = PodmanRemote.setup_for(
                 remote_resource, registries_authfile=get_authfile_path(config.registry)
             )
+            module_name = self.task_name + '_' + platform
+            tracer = get_tracer(module_name=module_name, service_name=OTEL_SERVICE_NAME)
+            with tracer.start_as_current_span("build_container") as span:
+                span.set_attribute('git_ref', self._params.user_params.get('git_ref'))
+                span.set_attribute('git_uri', self._params.user_params.get('git_uri'))
+                span.set_attribute('git_commit',
+                                   self._params.source.provider_params.get('git_commit', None))
+                span.set_attribute('build_args', str(self.workflow_data.buildargs))
```

Comment:
build args are just args added for cachito, so it doesn't have any value

Reply:
removed